.github/workflows/feature.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build and Test
+on:
+  pull_request:
+    branches:
+      develop
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: Run Unit tests.
+        run: make test-coverage
+      - name: Upload Coverage report to CodeCov
+        uses: codecov/codecov-action@v1.0.0
+        with:
+          token: ${{secrets.CODECOV_TOKEN}}
```

Comment:
maybe we don't need this?

in https://github.com/codecov/codecov-action#usage:
> token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos

Reply:
As it's failing: https://github.com/mkfsn/mizukinana/pull/7/checks?check_run_id=886513220, reverting this change.